cellpack/autopack/Environment.py

```diff
@@ -66,8 +67,17 @@
 from panda3d.bullet import BulletRigidBodyNode
 
 import cellpack.autopack as autopack
+from cellpack.autopack.Grid import Grid
 from cellpack.autopack.MeshStore import MeshStore
+from cellpack.autopack.ingredient import Ingredient
 from cellpack.autopack.loaders.recipe_loader import RecipeLoader
+from cellpack.autopack.utils import (
+    cmp_to_key,
+    deep_merge,
+    ingredient_compare0,
+    ingredient_compare1,
+    ingredient_compare2,
```

Comment:
Is there a way to make the difference between these functions more clear in the name?